Etch.OrchardCore.Favicon.csproj

```diff
@@ -13,13 +13,25 @@
   </PropertyGroup>
 
   <ItemGroup>
-    <PackageReference Include="OrchardCore.ContentFields" Version="1.4.0" />
-    <PackageReference Include="OrchardCore.ContentManagement" Version="1.4.0" />
-    <PackageReference Include="OrchardCore.DisplayManagement" Version="1.4.0" />
-    <PackageReference Include="OrchardCore.Media" Version="1.4.0" />
-    <PackageReference Include="OrchardCore.Navigation" Version="1.4.0" />
-    <PackageReference Include="OrchardCore.ResourceManagement" Version="1.4.0" />
-    <PackageReference Include="OrchardCore.Module.Targets" Version="1.4.0" />
+    <EmbeddedResource Remove=".gitignore" />
+    <EmbeddedResource Remove=".prettierrc" />
```

Comment:
`.prettierrc`, `package.json`, `package-lock.json` & `webpack.config.js` don't exist in this project, they can be removed.